### ISSUE TITLE
openapi-django branch should be merged into master

### ISSUE THREAD

**Issue by u8sand** [Nitrogen-DCPPC/FAIRshake#21]
This is difficult because it's essentially an unrelated tree but the other branch solves all the issues marked `done-awaiting-merge`. The primary things that need to be done before they can be merged are:

- [x] Completely functioning UI
- [x] Old -> New Database Migration
- [x] Dependencies must adopt the new API
  - [x] Chrome Extension
  - [x] Bookmarklet

**Comment by u8sand** [comment]
@lw453 Please see to what extent the chrome extension and bookmarklet can be modified to utilize the current database and client library (`insignia.js`).
